.circleci/config.yml

```diff
@@ -514,7 +514,7 @@ executors:
         type: string
         default: medium
     docker:
-      - image: mbgl/android-ndk-r21e:latest
+      - image: mbgl/android-ndk-r22:latest
```

Comment:
is this required? we need to bump ndk version at the same time with all the projects we have, and start from common.

Reply:
you all are correct that it would be best to add `android-31` to `r21e`. resolving this thread in favor of this one https://github.com/mapbox/mapbox-maps-android/pull/1004#discussion_r787692985